emsize.py

```diff
@@ -37,32 +37,47 @@ def error(text):
 
 
 def parse_args(argv):
-  parser = argparse.ArgumentParser(description=__doc__)
-  parser.add_argument('-format', '--format')
-  parser.add_argument('file')
-  args = parser.parse_args(argv)
-  return args.file
+  files = []
+  params = []
+  appendFormat = True
+  ret = 0
+  for arg in argv:
```

Comment:
It looks like the reason you removed `argparse` and wrote your own parser is to pass through unknown args to llvm-size, right? Did you try using argparse's [parse_known_args](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_known_args) for this? If that works, it would be better than having our own argument parsing.